src/server/command/worldedit/replace.zig

```diff
@@ -0,0 +1,61 @@
+const std = @import("std");
+
+const main = @import("main");
+const Vec3i = main.vec.Vec3i;
+const User = main.server.User;
+
+const Block = main.blocks.Block;
+const Blueprint = main.blueprint.Blueprint;
+const Pattern = main.blueprint.Pattern;
+const Mask = main.blueprint.Mask;
+
+pub const description = "Replace all blocks matching <old> mask expression with blocks from <new> block pattern.";
+pub const usage = "/replace <old> <new>";
```

Comment:
```suggestion
pub const description = "Replace blocks in the worldedit selection box.";
pub const usage = "/replace <old mask> <new pattern>";
```

Reply:
Agreed except "box", that is overspecification, there is only one world edit selection and in future it may no longer be a box.